src/components/modes/teacher/BackgroundToggle.js

```diff
@@ -90,7 +97,10 @@ const BackgroundToggle = () => {
         control={
           <Switch
             color="primary"
-            checked={backgroundToggleSetting?.data?.toggle || DEFAULT_BACKGROUND_ENABLED}
+            checked={
+              backgroundToggleSetting?.data?.toggle ||
```

Comment:
?? instead of || 

It doesn't show up since your default value is false, but if `toggle` is false it will default to the default value. `??` check for nullity so if toggle is false, it will use this value.